packages/core/src/views/account-center/Minimized.svelte

```diff
@@ -221,23 +215,32 @@
         })`}
       >
         <div class="flex items-center">
-          <div
-            class:color-yellow={!validAppChain}
-            class:color-white={validAppChain && !validAppChain.icon}
-            class="chain-icon flex justify-center items-center"
-            style={`background-color: ${
-              validAppChain
+          <div class="chain-icon-container">
```

Comment:
Does the same thing need to happen in Micro.svelte?

Reply:
@leightkt good question! Luckily micro is already using the WalletAppBadge component that handles the chain icons properly